src/offline.cpp

```diff
@@ -131,38 +131,14 @@ QDBusPendingReply<> Offline::triggerUpgrade(Action action)
     return QDBusConnection::systemBus().asyncCall(msg, 24 * 60 * 1000 * 1000);
 }
 
-QDBusPendingReply<bool> Offline::getResult()
+QDBusPendingReply<bool, QStringList, Transaction::Role, qint64, Transaction::Error, QString> Offline::getResults()
 {
     // Manually invoke dbus because the qdbusxml2cpp does not allow
     // setting the ALLOW_INTERACTIVE_AUTHORIZATION flag
     auto msg = QDBusMessage::createMethodCall(PK_NAME,
                                               PK_PATH,
                                               PK_OFFLINE_INTERFACE,
-                                              QStringLiteral("GetResult"));
-    msg.setInteractiveAuthorizationAllowed(true);
-    return QDBusConnection::systemBus().asyncCall(msg, 24 * 60 * 1000 * 1000);
-}
-
-QDBusPendingReply<QStringList> Offline::getResultPackages()
```

Comment:
Why did you remove this method? not up2date with current API but this is now an undefined method since it still in the .h

Reply:
That one was merged too quickly before the upstream API was finalized, so editing it is good actually...

Reply:
Good catch, I fixed the header too.